lib/constants.js

```diff
@@ -0,0 +1,132 @@
+ *     DIRECTORY_CONF: string, DIRECTORY_LIB: string, DIRECTORY_MODULES: string, FILE_APP_CONF:
+ *     string, FILE_UUF_CONF: string, MODULE_AUTH: string, MODULE_HANDLEBARS: string, MODULE_LESS:
+ *     string, APP_CONF_APP_NAME: string, APP_CONF_CACHE_ENABLED: string,
+ *     APP_CONF_DEBUGGING_ENABLED: string, APP_CONF_PERMISSION_ROOT: string,
+ *     APP_CONF_ADMIN_SERVICES_URL: string, APP_CONF_LOGIN_PAGE: string, APP_CONF_ERROR_PAGES:
+ *     string, APP_CONF_AUTH_MODULE: string, APP_CONF_AUTH_MODULE_ENABLED: string,
+ *     APP_CONF_AUTH_MODULE_TRANSPORT: string, APP_CONF_AUTH_MODULE_LOGIN: string,
+ *     APP_CONF_AUTH_MODULE_LOGIN_ON_SUCCESS: string, APP_CONF_AUTH_MODULE_LOGIN_ON_SUCCESS_SCRIPT:
+ *     string, APP_CONF_AUTH_MODULE_LOGIN_ON_SUCCESS_PAGE: string,
+ *     APP_CONF_AUTH_MODULE_LOGIN_ON_FAIL: string, APP_CONF_AUTH_MODULE_LOGIN_ON_FAIL_SCRIPT:
+ *     string, APP_CONF_AUTH_MODULE_LOGIN_ON_FAIL_PAGE: string, APP_CONF_AUTH_MODULE_LOGOUT:
+ *     string, APP_CONF_AUTH_MODULE_LOGOUT_ON_SUCCESS: string,
+ *     APP_CONF_AUTH_MODULE_LOGOUT_ON_SUCCESS_SCRIPT: string,
+ *     APP_CONF_AUTH_MODULE_LOGOUT_ON_SUCCESS_PAGE: string, APP_CONF_AUTH_MODULE_LOGOUT_ON_FAIL:
+ *     string, APP_CONF_AUTH_MODULE_LOGOUT_ON_FAIL_SCRIPT: string,
+ *     APP_CONF_AUTH_MODULE_LOGOUT_ON_FAIL_PAGE: string, APP_CONF_AUTH_MODULE_SSO: string,
+ *     APP_CONF_AUTH_MODULE_SSO_ENABLED: string, APP_CONF_AUTH_MODULE_SSO_ISSUER: string,
+ *     APP_CONF_AUTH_MODULE_SSO_RESPONSE_SIGNING_ENABLED: string,
+ *     APP_CONF_AUTH_MODULE_SSO_KEY_STORE_NAME: string,
+ *     APP_CONF_AUTH_MODULE_SSO_KEY_STORE_PASSWORD: string,
+ *     APP_CONF_AUTH_MODULE_SSO_IDENTITY_PROVIDER_ALIAS: string,
+ *     APP_CONF_AUTH_MODULE_SSO_IDENTITY_PROVIDER_URL: string,
+ *     APP_CONF_AUTH_MODULE_SSO_INTERMEDIATE_PAGE: string, UUF_CONF_DISPLAY_NAME: string,
+ *     UUF_CONF_LOG_LEVEL: string, UUF_CONF_ERROR_PAGES: string, UUF_CONF_SECURITY_CONSTRAINTS:
+ *     string, UI_COMPONENT_JS_FUNCTION_ON_REQUEST: string, UI_COMPONENT_DEFINITION_VERSION:
+ *     string, UI_COMPONENT_DEFINITION_INDEX: string, UI_COMPONENT_DEFINITION_EXTENDS: string,
+ *     UI_COMPONENT_DEFINITION_PERMISSIONS: string, UI_COMPONENT_DEFINITION_DISABLED: string,
+ *     PAGE_DEFINITION_URI: string, PAGE_DEFINITION_LAYOUT: string, PAGE_DEFINITION_IS_ANONYMOUS:
+ *     string, UNIT_DEFINITION_PUSHED_URIS: string, HELPER_PARAM_PARAMS: string,
+ *     HELPER_PARAM_OVERRIDE: string, HELPER_PARAM_COMBINE: string, HELPER_PARAM_SCOPE: string,
+ *     COMBINED_RESOURCES_SEPARATOR: string, COMBINED_RESOURCES_URL_TAIL: string,
+ *     CACHE_KEY_UUF_CONF: string, CACHE_KEY_UUF_CONF_FILE_LMD: string, CACHE_KEY_APP_CONF: string,
+ *     CACHE_KEY_APP_CONF_FILE_LMD: string, CACHE_KEY_LOOKUP_TABLE: string, CACHE_KEY_USER: string,
+ *     CACHE_KEY_SSO_SESSIONS: string, CACHE_KEY_HANDLEBARS_ROOT: string, URL_PARAM_REFERER:
+ *     string}}
+ */
+var constants = {
```

Comment:
Used simple 'c' here by following the naming convention used for modules. See [/lib/dynamic-files-renderer.js#L8](https://github.com/sajithar/carbon-uuf/blob/master/lib/dynamic-files-renderer.js#L8), [/lib/utils.js#L7](https://github.com/sajithar/carbon-uuf/blob/master/lib/utils.js#L7), [/lib/error-pages-router.js#L1](https://github.com/sajithar/carbon-uuf/blob/master/lib/error-pages-router.js#L1) & etc.